skyrim-platform/src/platform_se/codegen/TSConverter.cpp

```diff
@@ -0,0 +1,229 @@
+
+#include <iostream>
+#include <iostream> // For stringstream
+#include <filesystem> // For exist()
+#include <iomanip> // For setw() 
+#include <cctype> // for toupper() and tolower()
+#include <fstream> // For ifstream and ofstream
+#include <functional> // For std::function
+#include <algorithm> // For transform
+#include <set>
+#include "nlohmann/json.hpp"
+
+int main(int argc, char* argv[])
+{
+#ifdef _DEBUG
```

Comment:
As codegen runs for both configurations (Release/Debug), it should behave the same for all configurations.